synapse/config/experimental.py

```diff
@@ -139,3 +139,6 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
 
         # MSC3391: Removing account data.
         self.msc3391_enabled = experimental.get("msc3391_enabled", False)
+
+        # MSC3925: do not replace events with their edits
+        self.msc3925_inhibit_edit = experimental.get("msc3925_inhibit_edit", False)
```

Comment:
Oh the double negative is getting to me, we default this to `False` and then later check `not self._inhibit_replacement_via_edits` (== defaulting to `True`), so this will keep the same behavior as today. 👍